dipy/align/sumsqdiff.pyx

```diff
@@ -91,12 +113,35 @@ cdef int _solve_3d_symmetric_positive_definite(double[:] g, double[:] y, double
     return 0
 
 
-def solve_3d_symmetric_positive_definite(double[:] g, double[:] y, double tau,
-                                         double[:] out):
+def solve_3d_symmetric_positive_definite(g, y, double tau):
     r"""
     Wrapper for _solve_3d_symmetric_positive_definite (see documentation above)
```

Comment:
Same applies here (and you would probably want to edit it in a corresponding manner anyway)